modules/ROOT/pages/comments-callback-mode.adoc

```diff
@@ -24,7 +24,7 @@ All options accept functions incorporating `+done+` and `+fail+` callbacks as pa
 
 Most (create, reply, and edit) functions require an `+id+` identifying the *current author*.
 
-Current author : The Comments plugin does not know the name of the current user. Determining the current user and storing the comment related to that user, has to be configured by the developer.
+Current author:: The Comments plugin does not know the name of the current user. Determining the current user and storing the comment related to that user, has to be configured by the developer.
```

Comment:
@lnewson double colon?

Reply:
Yep, if you look at the 5.x docs they are meant to be description lists instead in this case. See https://docs.asciidoctor.org/asciidoc/latest/lists/description/